docs/tools/flow-cli/accounts/account-fund.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Funding a Testnet Account
+description: How to fund a Testnet Flow account from the command line
+sidebar_position: 7
+---
+
+Fund a valid Testnet Flow Account using the Flow CLI.
```

Comment:
Think this could be expanded on a little. E.g. for people with no context mention faucet, what it is, faucet link, etc

Reply:
updated with a info box with some information about the faucet